docs/development/extensions-core/datasketches-theta.md

```diff
@@ -95,6 +95,49 @@ This returns a summary of the sketch that can be used for debugging. This is the
 }
 ```
 
+
+
+### Constant Theta Sketch 
+
+This post aggregator allows the user to add a constant theta sketch which can be used in other post-aggregators e.g. `thetaSketchSetOp` 
+
+```json
+{
+  "type"  : "thetaSketchConstant",
+  "name": <output name>,
+  "value"  : <base64-encoded value of sketch>
+}
+```
+
+### Example of Constant Theta Sketch 
+
+Assume we have a datasource of a variety of users. Using `filters` and `aggregation`, we generate a theta sketch of all `football fans`.  
+
+We have a third-party provider who has provided us a constant theta sketch of all `cricket fans`. We would like to `INTERSECT` both in a `post-aggregation` stage to identify users who are interested in both `cricket` and `football` and then use `thetaSketchEstimate` to calculate the number of unique users.
```

Comment:
```suggestion
A third-party provider has provided a constant theta sketch of all `cricket fans` and you want to `INTERSECT` both cricket fans and football fans in a `post-aggregation` stage to identify users who are interested in both `cricket`. The you want to use `thetaSketchEstimate` to calculate the number of unique users.
```

Awesome example. I think it might be helpful to show an example of the final output to help visualize the final result of the operation.